clusterloader2/pkg/config/cluster.go

```diff
@@ -34,6 +34,7 @@ type ClusterConfig struct {
 	KubeConfigPath             string
 	Nodes                      int
 	Provider                   string
+	EtcdInsecurePort           string
 	MasterIPs                  []string
```

Comment:
Use IntEnvVar flag - ports can be represented as integers. Change type of port to int throughout your PR.

Reply:
done